### PR TITLE
New stats icon to fix previous off-centered icon. Possible bug fix in…

### PR DESCRIPTION
**Pull request by cmenkemeller**
… attraction_list_widget?

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -18,6 +18,8 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^0.1.2
+  # Used for multiple icon sets, including Entypo
+  flutter_icons:
```
Comment (thomasstoeckert) [inline pubspec.yaml]: No set version for this package? I could have sworn that's required, but IDK.

Comment (cmenkemeller) [reply pubspec.yaml]: Runs without but I'll add it just in case

File: lib/widgets/attractions_page/attraction_list_widget.dart [hunk 2]
```diff
@@ -100,7 +100,7 @@ class _AttractionsListViewState extends State<AttractionsListView> {
     if (_hasDefunct && widget.parentPark.showDefunct) {
       headedList.add("Defunct");
       headedList.addAll(defunctList);
-      fullList.addAll(seasonalList);
+      fullList.addAll(defunctList);
```
Comment (thomasstoeckert) [inline lib/widgets/attractions_page/attraction_list_widget.dart]: The `fullList` is an artifact from when I was playing around with headers and such. This is a proper fix, if we were using it, but I left it all in forgetfully, so it doesn't solve anything. Actually, could you remove all code pertaining to the fullList for me? Thanks.

File: lib/ui/home.dart [hunk 3]
```diff
@@ -499,7 +500,7 @@ class _HomePageState extends State<HomePage> {
                               db: widget.db,
                               serverParks: _parksManager.allParksInfo,
                             ))),
-                    child: _buildMenuIcon(FontAwesomeIcons.chartPie))
+                    child: _buildMenuIcon(Entypo.getIconData("pie-chart")))
```
Comment (thomasstoeckert) [inline lib/ui/home.dart]: Like the new icon 👍

Comment (thomasstoeckert) [reply lib/ui/home.dart]: Also, it looks like FontAwesomeIcons is included with flutter_icons. If possible, could you remove the old FontAwesomeIcons package and update all references to it to the new flutter_icons library? That way we don't have to have FontAwesome in there twice.

File: lib/widgets/attractions_page/attraction_list_widget.dart (hunk 2 above)
Comment (cmenkemeller) [reply lib/widgets/attractions_page/attraction_list_widget.dart]: Tried updating the icons. Some of them (clocks, circles, medal) are not transitioning very well. The vector images via this new package appear to be older (ex FontAwesome is v4.7.0 vs the current 8.2.0) . Alternatives?

**Pull request merged**